### PR TITLE
{devel}[system/system] Flink v1.11.2

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-042 - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/3d9e48d60ee2e14b84d0e5110b39b209 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/81ae306ed1e0cfc3c664a45eb7fcced4 for a full test report.

File: easybuild/easyconfigs/f/Flink/Flink-1.11.2-bin-scala_2.11.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'Flink'
+version = '1.11.2'
+versionsuffix = '-bin-scala_2.11'
```
Comment (lexming) [inline easybuild/easyconfigs/f/Flink/Flink-1.11.2-bin-scala_2.11.eb:5]:
```suggestion
local_scala_version = '2.11'
versionsuffix = '-bin-scala_%s' % local_scala_version
```

File: easybuild/easyconfigs/f/Flink/Flink-1.11.2-bin-scala_2.11.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+homepage = 'https://flink.apache.org/'
+description = """Apache Flink is a framework and distributed processing engine for stateful computations
+over unbounded and bounded data streams.
+Flink has been designed to run in all common cluster environments, perform computations at in-memory speed
+and at any scale."""
+
+toolchain = SYSTEM
+
+sources = ['%(namelower)s-%(version)s-bin-scala_2.11.tgz']
```
Comment (lexming) [inline easybuild/easyconfigs/f/Flink/Flink-1.11.2-bin-scala_2.11.eb:15]:
```suggestion
sources = ['%%(namelower)s-%s%s.tgz' % (version, versionsuffix)]
```

File: easybuild/easyconfigs/f/Flink/Flink-1.11.2-bin-scala_2.11.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+checksums = [
+    '63e89bba954963cde182d00a7ea3d1c24a90c764fd5342b6394bc0d888a634d8',  # flink-1.11.2-bin-scala_2.11.tgz
+]
+
+dependencies = [('Java', '1.8')]
+
+sanity_check_paths = {
+    'dirs': ['conf'],
+    'files': ['bin/flink']
+}
```
Comment (lexming) [inline easybuild/easyconfigs/f/Flink/Flink-1.11.2-bin-scala_2.11.eb:29]:
```suggestion
sanity_check_paths = {
    'files': ['bin/flink', 'lib/flink-dist_%s-%s.jar' % (local_scala_version, version)],
    'dirs': ['conf', 'opt', 'plugins'],
}

sanity_check_commands = ['flink --version']
```

**Review by lexming — CHANGES_REQUESTED** [review]
Thanks for this PR, it works well. I would like to make a minor improvement regarding the sanity checks. Please consider my suggestions below

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node378.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3320f57efc3dfc8a500d6ffc4de4816f for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node128.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/94262fc47b20c9eb67f3187fa1c3f468 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @Falkor !

**Pull request merged**